download.ps1

```diff
@@ -0,0 +1,48 @@
+$DataDir = "./data"
+$SenaFile = "$DataDir\sena-3.zip"
+$ElawaFile = "$DataDir\elawa.zip"
+$SenaChecksum = "BEC5131799DB07BF8D84D8FC1F3169FB2574F2A1F4C37F6898EAB563A4AE95B8"
+$ElawaChecksum = "E3608F1E3188CE5FDB166FBF9D5AAD06558DB68EFA079FB453881572B50CB8E3"
```

Comment:
As mentioned elsewhere, the filename and checksum values here should be script parameters, which Taskfile.yml will pass to the download.ps1 script.